tests/dragonfly/connection_test.py

```diff
@@ -281,7 +281,7 @@ def generate(max):
 
 
 @pytest.mark.asyncio
```

Comment:
why you did not remove the asyncio mark here?

Reply:
We have yet to do a proper clean-up for all the marks - just found a way to skip them